### ISSUE TITLE
Uppercase field validation message

### ISSUE THREAD

**Issue by Tarasovych** [laravel/framework#28946]
- Laravel Version: 5.8.17
- PHP Version: 7.2.14-1+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Jan 13 2019 10:05:45) ( NTS )
- Database Driver & Version: not involved

### Description:
When there is an uppercase FormRequest field and validation error occurs, field name will be lowercased and splitted by space in the error message.

### Steps To Reproduce:
1. Clone [repository](https://github.com/Tarasovych/formrequest-issue)
2. Navigate to cloned folder
3. Run `composer install`
4. Open `welcome` route.
5. Press "Submit" button.
6. See "The u p p e r c a s e field is required."

Expected result: "The UPPERCASE field is required." should be shown.

TL;DR:
![image](https://user-images.githubusercontent.com/24782123/60080994-9a598980-9739-11e9-8be5-7269a7d9bb62.png)
![image](https://user-images.githubusercontent.com/24782123/60082654-db9f6880-973c-11e9-9c91-0b76b8705476.png)

The result of [this](https://github.com/laravel/framework/blob/5.8/src/Illuminate/Support/Str.php#L495) row is `u_p_p_e_r_c_a_s_e`

**Comment by bonzai** [comment]
Laravel uses `Str::snake` to [format attributes](https://github.com/laravel/framework/blob/5.8/src/Illuminate/Validation/Concerns/FormatsMessages.php). You can set a custom attribute name by overriding the attributes method:

https://laravel.com/docs/5.8/validation#customizing-the-validation-attributes

**Comment by ManojKiranA** [comment]
```/**
     * Get custom attributes for validator errors.
     *
     * @return array
     */
    public function attributes(): array
    {
        return [
            'UPPERCASE' => 'Email Address',
        ];
    }```

@bonzai As you said `Str::snake('UPPERCASE')` 
Will result in   
`u_p_p_e_r_c_a_s_e`
But Not
`'u p p e r c a s e'`

```
Route::get('/snakeTest/',function(){
    $str = app('Illuminate\Support\Str');
    dd($str::snake('UPPERCASE'));
});
```

So Its not the Expected Result

**Comment by bonzai** [comment]
@ManojKiranA: Underscores will be replaced with spaces in `getDisplayableAttribute`:

https://github.com/laravel/framework/blob/5.8/src/Illuminate/Validation/Concerns/FormatsMessages.php#L256

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repo you can try one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by Tarasovych** [comment]
> it looks like this is a question

Sorry, but have I asked one?
